### PR TITLE
Axios requests

### PR DESCRIPTION
**Pull request by Oluane**

### CODE REVIEW

File: frontend/src/components/CarCard.jsx [hunk 1]
```diff
@@ -30,7 +31,7 @@ function CarCard(props) {
           {userVehicle.brand} | {userVehicle.model}
         </h1>
         <h3>
-          {userVehicle.enginePower} ({userVehicle.horsePower} CH)
+          {userVehicle.enginePower} ({userVehicle.horsePower} CH){" "}
```
Comment (laurentmahieu) [inline frontend/src/components/CarCard.jsx]: delete {" "}

File: frontend/src/pages/Login.jsx [hunk 2]
```diff
@@ -21,6 +21,11 @@ function Login(props) {
       .then(({ data }) => {
         history.push("/");
         props.dispatch({ type: "FETCHING_USER_DATA", value: { data } });
+        let userId = data.user.id;
+        axios.get(`${apiSite}/user/${userId}/vehicle`).then(({ data }) => {
+          console.log(data);
```
Comment (laurentmahieu) [inline frontend/src/pages/Login.jsx]: no console.log()

**Review by laurentmahieu — DISMISSED** [review]
good but no conole.log() please

**Review by laurentmahieu — DISMISSED** [review]

File: backend/route/vehicule.js [hunk 3]
```diff
@@ -56,6 +75,7 @@ router.get("/:id/historique", (req, res) => {
       } else {
         // Si tout s'est bien passé, on envoie le résultat de la requête SQL en tant que JSON.
         res.json(results);
+
```
Comment (LoicBrassart) [inline backend/route/vehicule.js]: (This empty line makes me think maybe Prettier is not properly used on your machine. If you need help to check it, feel free to poke me)

File: frontend/src/components/CarCard.jsx [hunk 4]
```diff
@@ -10,6 +10,7 @@ function CarCard(props) {
   const dispatch = useDispatch();
   const userVehicle = useSelector(state => state.user.carData);
 
+  // // useEffect et axios à retirer quand les routes sécurisées seront en place
```
Comment (LoicBrassart) [inline frontend/src/components/CarCard.jsx]: Don't leave instructions to future-you : either do it now, or add the task to your backlog

File: frontend/src/components/CarCard.jsx (hunk 1 above)
Comment (LoicBrassart) [inline frontend/src/components/CarCard.jsx]: (Does this new `{" "}` serve a purpose, or is it an artefact ? If it is the latter, please remove it)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some minor Evil, and questions

File: backend/route/vehicule.js (hunk 3 above)
Comment (Oluane) [reply backend/route/vehicule.js]: The request wasn't closed, so prettier didn't worked. But don't know why braces were missing, maybe due to a merge with dev ?

File: frontend/src/components/CarCard.jsx (hunk 1 above)
Comment (Oluane) [reply frontend/src/components/CarCard.jsx]: Prettier transforms the space between ")" and next variable with that when I save. But we need that space between the two.

**Review by laurentmahieu — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**